md/themes.md

```diff
@@ -28,7 +28,28 @@ Any CSS, images or JavaScript files can be shared using a theme resource.
 
 Export the examples to see how to structure a theme.
 
-The default themes are based on the [bootswatch example](https://bootswatch.com/), so you can easily choose another existing theme and package it to define a new theme. 
+### Using bootswatch for a custom theme
+
+The default themes are based on the [bootswatch example](https://bootswatch.com/), so you can easily choose another existing theme and package it to define a new theme.
+
+A second way to create a custom theme is starting from an new custom theme in Studio and using a bootswatch default sass theme. In order to do this using the 3.3.4 version of bootswatch:
+* In the custom theme file `package.json`, add `bootswatch` in the `devDependencies` section
+```json
+    "bootswatch": "3.3.4"
+```
+* In this same file `package.json`, include the bootswatch path towards the default theme in the scripts/build section   
+```json
+    --include-path ./node_modules/bootswatch/`name-of-default-theme`/
+```
+* In the file `src/scss/main.scss`, import the variables of the theme before importing bootstrap by doing: 
+```scss
+    @import "variables";
+```
+* In the file `src/scss/main.scss`, import bootswatch after importing bootstrap by doing: 
+```scss
+    @import "bootswatch";
+```
+* You can optionally remove the custom `src/scss/_bonita_buttons.scss` and `src/scss/_bonita_pager.scss` files, as well as everything except the font paths from the `src/scss/_bonita_variables.scss`. If you remove the custom files, also remove the imports of the files in the `src/scss/main.scss` file.
```

Comment:
- remove "the" (form `src/scss/_bonita_variables.scss`)
- "in `src/scss/main.scss`" or "in the file `src/scss/main.scss`"